webpack.config.js

```diff
@@ -0,0 +1,64 @@
+
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const path = require('path')
+
+module.exports = [{
+  mode: 'development',
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'chat.bundle.js'
+  },
+  plugins: [
+    new HtmlWebpackPlugin({ template: './index.html' })
+  ],
+  externals: {
+    fs: 'null',
+    'node-fetch': 'fetch',
+    'isomorphic-fetch': 'fetch',
+    xmldom: 'window',
+    'text-encoding': 'TextEncoder',
+    'whatwg-url': 'window',
+    '@trust/webcrypto': 'crypto'
+  },
+  devServer: {
+    contentBase: './dist'
+  },
+  devtool: 'source-map'
+},
+{
+  mode: 'development',
+  entry: './src/shortChatPane.js',
```

Comment:
I think you can simplify the last two webpack config objects by using an object in the entry, e.g. 

```
entry: {
  longChatPane: './src/longChatPane.js',
  shortChatPane: './src/shortChatPane.js'
},
output: {
  path: path.resolve(__dirname, 'dist'),
  filename: '[name].js'
},
```

Reply:
Done! thanks.